GitUI/UserControls/RevisionGridClasses/DvcsGraph.cs

```diff
@@ -179,7 +187,7 @@ public void ShowAuthor(bool show)
             this.DateColumn.Visible = show;
         }
 
-        [DefaultValue(FilterType.None)]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2002:DoNotLockOnObjectsWithWeakIdentity"), DefaultValue(FilterType.None)]
```

Comment:
I decided to take an opposite approach. I keep my suppress warning attribute and created a separate issue to review that and fix properly #2748